src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -109,7 +109,7 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             var legacySettings = await settingsProvider.GetAsync(oldBinding.ProjectKey);
 
             // TODO: add proper progress messages.
-            progress?.Report(new MigrationProgress(0, 1, "Getting files ...", false));
+            progress?.Report(new MigrationProgress(0, 1, "Getting files to clean...", false));
```

Comment:
Maybe `Finding files to clean ...`?

The other messages have a space before the `...` - it looks a bit on the screen, I think.